x-pack/agent/pkg/agent/application/fleet_gateway.go

```diff
@@ -101,17 +110,22 @@ func (f *fleetGateway) worker() {
 }
 
 func (f *fleetGateway) execute() (*fleetapi.CheckinResponse, error) {
-	// TODO(ph): Aggregates and send events.
-	cmd := fleetapi.NewCheckinCmd(f.agentInfo, f.client)
+	// get events
+	ee, ack := f.reporter.Events()
```

Comment:
Let's create a followup issue concerning batching, @nchaulet can you give us an idea of the maximum payload in the body we can send per request to the Kibana or maybe a sweet spot with performance/size? IIRC hapijs has a low value for the default, 1mb?

Reply:
it's the same value by default for kibana: 1mb